test/tools/go.mod

```diff
@@ -8,7 +8,6 @@ go 1.22.0
 require (
 	github.com/cpuguy83/go-md2man/v2 v2.0.6
 	github.com/go-swagger/go-swagger v0.30.5
-	github.com/onsi/ginkgo/v2 v2.22.1
```

Comment:
you also have to remove the ginkgo build references from test/tools/Makefile

Reply:
Done